### DISCUSSION TITLE
This is the GOAT (appreciation post)

### DISCUSSION THREAD

**Discussion by jbendy** [builtbybel/Bloatynosy#98] [General]
Just wanted to say THANK YOU to the creator for this amazing tool. You've improved my life so much! Thank you!!!

**Comment by 1mrblue** [comment]
I recently reinstalled Windows on my Surface Laptop and tried this. And it is amazing! Thank you for this, will pass this on to the rest of the people needing to debloat their systems.

**Comment by DeBl0at** [comment]
Thank you so much! My Surface Pro7 and Surface Go3 have now both got much better battery performance thanks to this amazing debloat. I am also feeling better in myself knowing that I have more control now of telemetry I was not asked my consent for. It is great to feel less of  a victim of multinational corporate privacy invasion. Power to the people!
